src/main/java/com/google/ehub/utility/NextIncreasingLexicographicStringUtility.java

```diff
@@ -0,0 +1,37 @@
+package com.google.ehub.utility;
+
+/**
+ * Utility class that holds method to find the next lexicographic string.
+ */
+public final class NextIncreasingLexicographicStringUtility {
+  /**
+   * Finds the next increasing lexicographic string.
+   *
+   * @param str the string used to find the next increasing lexicographic string
+   * @return the next lexicographic string in increasing order
+   */
+  public static String getNextIncreasingLexicographicString(String str) {
+    if (str == null) {
+      return "";
+    }
+    if (str.isEmpty()) {
+      return "a";
+    }
+
+    int i = str.length() - 1;
```

Comment:
let's use a better var name than "i"

Reply:
I found a better variable name and added the change in last commit.